highlight/RON.sublime-syntax

```diff
@@ -6,6 +6,10 @@ file_extensions:
 scope: source.ron
 contexts:
   main:
+    # extensions
```

Comment:
This could be done better, but I think it's sufficient. Could you try it out with your editor?

Reply:
I tested it. Seems to work fine.